.gitlab/tests/shared_flux_clusters.yml

```diff
@@ -86,7 +86,7 @@ run_tests_flux_tuolumne:
     matrix:
       - HOST: tuolumne
         ARCHCONFIG: llnl-elcapitan
-        BENCHMARK: [amg2023, kripke, laghos, raja-perf]
+        BENCHMARK: [amg2023, kripke, laghos, raja-perf, sparta-snl]
```

Comment:
Reverted fft_kokkos=hipfft here, as it is now set in package.py

Reply:
@michaelmckinsey1 `sparta/experiment.py` ensures that `fft_kokkos=hipfft` when building with `+rocm` (and `fft_kokkos=cufft` when building with `+cuda`). Hence, the default setting should work without any changes to the `package.py` itself